src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationContext.cs

```diff
@@ -0,0 +1,62 @@
+using System.Globalization;
+using System.Reflection;
+using Microsoft.Extensions.DependencyModel;
+
+namespace Serilog.Settings.Configuration;
+
+/// <summary>
+/// A context to adjust how the configuration object is processed.
+/// </summary>
+public class ConfigurationContext
```

Comment:
```suggestion
public sealed class ConfigurationContext
```

At least for the first time.